scheduler/core/components/selector/selector.py

```diff
@@ -364,8 +365,11 @@ def update_cc_and_iq(self,
         """
         if site not in self.collector.sites:
             raise ValueError(f'Selector update_cc_and_iq called with invalid site: {site.name}')
-        self.cc_per_site[site] = new_cc or Selector._default_cc
-        self.iq_per_site[site] = new_iq or Selector._default_iq
+
+        self.variant_per_site[site] = Variant(cc=new_cc or Selector._default_cc,
+                                              iq=new_iq or Selector._default_iq,
+                                              wind_spd=None,
+                                              wind_dir=None)
```

Comment:
`VariantSnapshot` but I think this should be handled since I got rid of the `cc` and `iq` dicts.